libtsuba/src/RDGSlice.cpp

```diff
@@ -57,12 +58,12 @@ unload_metadata_array(
     tsuba::RDGCore* core) {
   auto search_res = core->find_part_prop_info(array_name);
   if (!search_res) {
-    if (search_res.error() != tsuba::ErrorCode::NotFound) {
+    if (search_res.error() != tsuba::ErrorCode::PropertyNotFound) {
       return search_res.error().WithContext(
           "searching for array name in prop info list");
```

Comment:
Is this context useful?

Reply:
If that is the only place it can fail, maybe not.

Reply:
Without context, we get file and line number.

My rule of thumb is that if we are not printing a data item specific to that function, the context is not necessary and indeed is clutter.

Reply:
Seems like a sensible rule